core/src/main/java/com/graphhopper/routing/util/parsers/AbstractAccessParser.java

```diff
@@ -27,7 +27,7 @@
 
 public abstract class AbstractAccessParser implements TagParser {
     static final Collection<String> ONEWAYS = Arrays.asList("yes", "true", "1", "-1");
-    static final Collection<String> INTENDED = Arrays.asList("yes", "designated", "official", "permissive");
+    static final Collection<String> INTENDED = Arrays.asList("yes", "designated", "official", "permissive", "destination");
```

Comment:
`destination` can generally be considered as accessible and doesn't really matter for non-motorized transportation modes.